tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -2626,6 +2626,48 @@ def test_generate_messages_with_defaults():
             "ge_version": "0.14.7.manual_testing",
             "x-forwarded-for": "00.000.00.000, 00.000.000.000",
         },
+        {
```

Comment:
Additional payload to test the possibility of batch request being a `$variable` string.